src/utils/parameters.py

```diff
@@ -1,6 +1,10 @@
 import os
 import argparse
 
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
is this needed if its already loaded in `deadbot.py`?

Reply:
This time I can say for certain it is, though I'm still lost on how `load_dotenv` actually works. I was working on my main computer, saw I had a weird error after adding parameters, changed nothing except load_dotenv, and suddenly it worked.

Reply:
Yeah that's odd, you'd think it doing it in the main file (deadbot.py), would do it, but if not then that's fine 👌